doc/quick-install.md

```diff
@@ -1,16 +1,25 @@
 # Jitsi Meet quick install
 
-This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 8 (Jessie) or later, and Ubuntu 14.04 or later are supported out-of-the-box.
+This guide helps you  ___host your own Jitsi server___. If you want to have a video conference without setting up any infrastructure, use https://meet.jit.si instead.
 
-Debian Wheezy and other older systems may require additional things to be done. Specifically for Wheezy, [libc needs to be updated](http://lists.jitsi.org/pipermail/users/2015-September/010064.html).
+## Requirements
 
-Also note that a recent default Ubuntu installation has only the `main` repository enabled, and Jitsi Meet needs packages from `universe`. Check your `/etc/apt/sources.list` file, and if `universe` is not present refer to [Ubuntu's documentation](https://help.ubuntu.com/community/Repositories/Ubuntu) on how to enable it. (Usually it amounts to copying the `main` lines and changing to `universe`.)
+The installation steps documented here are intended for Debian-based GNU/Linux systems.
 
-N.B.:
+Systems running the following versions are supported by this guide:
 
-a.) All commands are supposed to be run by root. If you are logged in as a regular user with sudo rights, please prepend ___sudo___ to each of the commands.
+1. Debian 8 (Jessie) or later
+1. Ubuntu 14.04 or later
 
-b.) You only need to do this if you want to ___host your own Jitsi server___. If you just want to have a video conference with someone, use https://meet.jit.si instead.
+To check the operating system available, you can run `cat /etc/issue` at the command-line.
+
+_Note_: All commands require superuser privileges. If you are logged in as a regular user, you may need to prepend `sudo` to each of the commands.
+
+### Platform-specific notes
+
+On Debian Wheezy, [libc needs to be updated](http://lists.jitsi.org/pipermail/users/2015-September/010064.html).
```

Comment:
This link (and the web server `lists.jitsi.org`) doesn't appear to be working.  Although it's not ideal, the solution which maintains the content most accurately may be to replace this with an Internet Archive link.

I considered pulling the contents of the archived post into the document here, but given that it's for a relatively old version of Debian (June, 2016), it is probably preferable to keep the quick install document short.